src/main.rs

```diff
@@ -157,6 +175,60 @@ impl<'a> RuntimeContext<'a> {
 }
 
 #[cfg(test)]
+mod full_run_tests {
+
+    use std::path::PathBuf;
+    use counter::file_handling;
+
+    #[test]
+    #[ignore]
```

Comment:
It's ignored precisely because it takes about a minute to run. You can run it with

cargo test -- --ignored

This is the idiomatic way Rust handles long running tests.